src/test/java/com/taskadapter/redmineapi/ProjectIntegrationIT.java

```diff
@@ -209,12 +211,71 @@ public void tryUpdateProjectWithLongHomepage() throws RedmineException {
         }
     }
 
-    // Redmine ignores this parameter for "get projects" request. see bug
-    // http://www.redmine.org/issues/8545
-    // The field is already accessible for a specific project for a long time (GET /projects/:id)
-    // but in the projects list (GET /projects) it's only on the svn trunk for now (Sep 8, 2014).
-    // It will be included in Redmine 2.6.0 which isn't out yet.
-    @Ignore
+    @Test
+    public void testUpdateTrackersBeforeRedmineUpdate() throws RedmineException {
+        //Trackers are undefined for new project beans, updating trackers on the project object should be allowed
+        Project projectToCreate = generateRandomProject();
+        assertEquals(0, projectToCreate.getTrackers().size());
+        
+        Collection<Tracker> trackers=new HashSet<>(Arrays.asList(TrackerFactory.create(1,"Bug")));
+        projectToCreate.addTrackers(trackers);
+        assertEquals(1, projectToCreate.getTrackers().size());
+        
+        projectToCreate.clearTrackers();
+        assertEquals(0, projectToCreate.getTrackers().size());
+    }
+    
+    @Test
+    public void testUpdateTrackersAfterRedimineUpdate() throws RedmineException {
+        //Prerequisite: verify that redmine server has at least 3 trackers (e.g. default configuration)
+        List<Tracker> availableTrackers=mgr.getIssueManager().getTrackers();
+        assertTrue("a minumum of 3 trackers should be configured in redmine", availableTrackers.size()>=3);
+
+        Project projectToCreate = generateRandomProject();
+        String createdProjectKey = null;
+        try {
+            //project created with default trackers has been tested by testCreateProject
+            //test override of default trackers on project creation
+            projectToCreate.clearTrackers();
+            Project createdProject = projectManager.createProject(projectToCreate);
+            createdProjectKey = createdProject.getIdentifier();
+            assertEquals(0, createdProject.getTrackers().size());
+           
+            //add single tracker
+            Collection<Tracker> trackers=new HashSet<Tracker>(Arrays.asList(availableTrackers.get(0)));
+            createdProject.addTrackers(trackers);
+            projectManager.update(createdProject);
+            createdProject=projectManager.getProjectByKey(createdProjectKey);
+            assertEquals(1, createdProject.getTrackers().size());
+
+            //add more than one tracker, it does not replace previous trackers
+            trackers=new HashSet<Tracker>(Arrays.asList(availableTrackers.get(1), availableTrackers.get(2)));
+            createdProject.addTrackers(trackers);
+            projectManager.update(createdProject);
+            createdProject=projectManager.getProjectByKey(createdProjectKey);
```

Comment:
let's not reuse vars once they are defined. use separate variables for readability instead

Reply:
Done, tests updated to fluent style